### DISCUSSION TITLE
How to config option of babel plugin in extraBabelPlugins (umi v3)

### DISCUSSION THREAD

**Discussion by le2tuan** [umijs/umi#9844] [General]
Hello guys, I'am working on this babel plugins https://babeljs.io/docs/en/babel-plugin-transform-react-jsx with some configs like 
```
plugins": [
    ["@babel/plugin-transform-react-jsx", {
      "pragma": "h",
      "pragmaFrag": "Fragment",
    }]
  ]
```

But I don't know how to declare this in umi v3, I am going use this property extraBabelPlugins , but it only accept Array[string] like that
```
  extraBabelPlugins: [
    "@babel/plugin-proposal-optional-chaining"
  ],
```

Please guide me how to fix it. Many thanks

**Comment by qiutian00** [comment]
Maybe use it  like this：
```
extraBabelPlugins: [
    "@babel/plugin-proposal-optional-chaining",
    ["@babel/plugin-transform-react-jsx", {
      "pragma": "h",
      "pragmaFrag": "Fragment",
    }]
  ],
```

**Reply by le2tuan** [reply]
thank your for your answer , i will try it now :bow:
